ui/src/pages/my-bookmarks.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import Head from 'next/head';
+
+import * as Interfaces from '@interfaces';
+import * as Components from '@components';
+import * as Layouts from '@layouts';
+import * as Helpers from '@helpers';
+import * as Config from '@config';
+import * as Types from '@types';
+import * as api from '@api';
+
+export const getServerSideProps: Types.GetServerSideProps = async (context) => {
+    // If not logged in, i.e no token with the right key, send them to ocrid to login and return
+    const token = Helpers.guardPrivateRoute(context);
+    // If logged in, grab the token from cookies
+
+    let user: Interfaces.User | null = null;
+
+    let usersBookmarks: Interfaces.BookmarkedPublicationsData[] = [];
+
+    try {
+        const response = await api.get(`${Config.endpoints.bookmarks}`, token);
+        usersBookmarks = response.data;
+    } catch (err) {
+        console.log('err');
```

Comment:
Not sure if the intention is to output the error itself: If so, just needs the quotes removing.

Reply:
Ah yes - my bad!